OpenDF-UI/app/containers/FileBrowser/index.js

```diff
@@ -9,32 +9,34 @@ import Helmet from 'react-helmet';
 import { FormattedMessage } from 'react-intl';
 import TreeView from '../../components/TreeView';
 import DetailView from '../../components/DetailView';
+import Paper from 'material-ui/Paper';
```

Comment:
Are we using `Paper?`

Reply:
Earlier we thought of using both Card and Paper here. But now since we are not using them here, we can remove those imports.